applications/launchpad/gui-react/src/styles/styles/typography.ts

```diff
@@ -13,64 +13,76 @@ const typography = {
     fontSize: 32,
     lineHeight: '44px',
     fontFamily: 'AvenirHeavy',
+    fontWeight: 500,
```

Comment:
shouldn't the fontWeight be the same as above? 800 for AvenirHeavy, 500 for AvenirMedium and 400 for AvenirRegular?
instead of 500 across the whole typography?

Reply:
Only `fontWeight: 500` renders the correct font if the `as` prop is provided in the `<Text />` component. Any other value doesn't work.